src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/UploadFileItem.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-list-item>
+    <v-row class="d-flex align-center">
+      <!-- File Name and Size -->
+      <v-col cols="4">
+        <div>{{ file.name }} ({{ (file.size / 1024).toFixed(2) }} KB)</div>
```

Comment:
same comment as before regarding files that are MB in size vs KB. 

also file name needs to be truncated when it's too long 
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/489c8eea-96e7-4cfb-ac76-3edc1ad62004)

could do this with css or a javascript function https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
<img width="1212" alt="Screenshot 2024-07-09 at 12 27 16 PM" src="https://github.com/bcgov/ECC-ECER/assets/10508230/df7f708e-9408-44cb-9885-551a06fb0c2d">